scripts/validate_rst_title_capitalization.py

```diff
@@ -21,147 +21,150 @@
 if TYPE_CHECKING:
     from collections.abc import Iterable
 
-
-CAPITALIZATION_EXCEPTIONS = {
-    "pandas",
-    "pd",
-    "Python",
-    "IPython",
-    "PyTables",
-    "Excel",
-    "JSON",
-    "HTML",
-    "SAS",
-    "SQL",
+CAPITALIZATION_EXCLUSIONS = {
```

Comment:
Proposed to change the name of this list from _EXCEPTIONS to _EXCLUSIONS, because exceptions seems to be more related to error handling in the code itself and this name may be confusing. 
However I'm happy to discuss or drop this change if you think it's out of scope of that story or is unnecessary